plugin-server/src/worker/ingestion/person-state.ts

```diff
@@ -90,6 +90,12 @@ export class PersonState {
         return this.personContainer
     }
 
+    async updateExceptProperties(): Promise<LazyPersonContainer> {
```

Comment:
Can you add a comment here that we do process properties: if we are merging persons or if this creates the person (it could be very confusing to read/debug otherwise later)